builtin/logical/database/path_config_connection.go

```diff
@@ -648,6 +648,14 @@ func (b *databaseBackend) connectionWriteHandler() framework.OperationFunc {
 				"Vault (or the sdk if using a custom plugin) to gain password policy support", config.PluginName))
 		}
 
+		// We can ignore the error at this point since we're simply adding a warning.
+		dbType, _ := dbw.Type()
+		if dbType == "snowflake" && config.ConnectionDetails["password"] != nil {
+			resp.AddWarning(`[DEPRECATED] Single-factor password authentication is deprecated in Snowflake and will
+be removed by November 2025. Please be ready for multi-factor authentication (MFA) configuration to continue use.
```

Comment:
> Please be ready for multi-factor authentication (MFA) configuration to continue use.

Is this potentially misleading or confusing? I don't think we have Snowflake MFA support on our roadmap so maybe we should not reference that?

Reply:
Yeah the snowflake docs I was looking at go over both key pair and MFA auth and I just accidentally wrote the wrong one, thanks for catching.